app.js

```diff
@@ -0,0 +1,205 @@
+const hamburger = document.querySelector('#icon');
+const navmenu = document.querySelector('nav');
+hamburger.addEventListener('click', () => {
+  hamburger.classList.toggle('fa-times');
+  navmenu.classList.toggle('nav-toggle');
+});
+
+window.addEventListener('scroll', () => {
+  hamburger.classList.remove('fa-times');
+  navmenu.classList.remove('nav-toggle');
+});
+const cards = [
+  {
+    image: 'images/budget.png',
+    title: 'Wishyacht',
+    info: ['frontend', 'backend', 'fullstack'],
+    description:
+    "a web application that allows users to rent and share their wishyacht's.",
+    skills: ['react', 'redux', 'rails'],
+  },
+  {
+    image: 'images/wishyacht.png',
+    title: 'Budget mobile',
+    info: ['Transactions', 'backend', '2022'],
+    description:
+      "a mobile application's that allows users to manage their budget and transactions.",
+    skills: ['ruby', 'rails', 'bootstrap'],
+  },
+  {
+    image: 'images/company.png',
+    title: 'Companies 3D',
+    info: ['frontend', 'Api', '2022'],
+    description:
+      'a web application that allows users to see the metrics of the top companies.',
+    skills: ['react', 'redux', 'bootstrap'],
+  },
+  {
+    image: 'images/space.png',
+    title: 'Space travellers',
+    info: ['pair', 'APIs', '2022'],
+    description:
+      'Basic app for doing some reservations.',
+    skills: ['react', 'redux', 'bootstrap'],
+  },
+];
+
+const work = document.querySelector('#portfolio');
+function cardsBuilder(card, index) {
+  const {
+    image, title, info, description, skills,
+  } = card;
+  const infoHtml = info.map(
+    (i) => `<li class="canopy">${i}</li>
+    ${
+  i === info[info.length - 1]
+    ? ''
+    : '<li><img src="images/Counter.png" alt="counter" /></li>'
+}`,
+  ).join('');
+  const skillsHtml = skills.map(
+    (s) => `<li class="highlight font-1">${s}</li>`,
+  ).join('');
+  work.innerHTML += `
+  <div class="card card-${(index === 0 || index % 2 === 0) ? '1' : '2'}">
+        <a href="#"
+          ><img src="${image}" alt="Snapshoot-Portfolio" class="snapshot"
+        /></a>
+        <div class="${(index === 0 || index % 2 === 0) ? 'right' : 'left'}-cont">
+          <h3 class="tonic flex0">${title}</h3>
+          <ul class="ul-box1 flex0">
+            ${infoHtml}
+          </ul>
+          <p class="description-2 flex0">
+            ${description}
+          </p>
+          <ul class="ul-box2 flex0">
+            ${skillsHtml}
```

Comment:
- [ ]  there is a tag that is overflowing please make sure to fix it in order to match the Figma design 
<img width="121" alt="Screen Shot 2022-10-20 at 17 09 42" src="https://user-images.githubusercontent.com/61790587/197002245-13ee7623-5d63-4e1b-8142-bf846c75c2dc.png">